src/html/verificationSuccessfull.html

```diff
@@ -125,11 +125,10 @@
             d="M61.5531 102.929L91.1762 125.94M91.1762 125.94L146.75 65.75L112.423 102.929L91.1762 125.94ZM99.5 5L5 59V68.72C4.9993 97.0344 14.2224 124.578 31.2733 147.183C48.3242 169.788 72.2748 186.222 99.5 194C126.725 186.222 150.676 169.788 167.727 147.183C184.778 124.578 194.001 97.0344 194 68.72V59L99.5 5Z"
             stroke="white"
             stroke-width="10"
-            stroke-linejoin="round"
-          />
+            stroke-linejoin="round" />
         </svg>
         <h1>Verification Successfull</h1>
-        <a href="https://portal.bashaway.sliitfoss.org"><button>Go to home</button></a>
+        <a href="%VERIFY_URL_PLACEHOLDER%"><button>Go to home</button></a>
```

Comment:
Shall we use the handlebars syntax here ? {{frontendDomain}} and use it during replacement so that everything looks the same just to keep it clean ?

Also there's a typo in this filename which I've missed, shall we correct it along with this. It should be `verificationSuccessful`